src/aap_eda/settings/post_load.py

```diff
@@ -0,0 +1,409 @@
+            "aap_eda.api.openapi.preprocess_filter_api_routes"
+        ],
+        "GENERIC_ADDITIONAL_PROPERTIES": "bool",
+    }
+
+
+def _get_logging_setup(settings: Dynaconf) -> dict:
+    return {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "simple": {
+                "format": "{asctime} {name} {levelname:<8} {message}",
+                "style": "{",
+            },
+        },
+        "handlers": {
+            "console": {
+                "class": "logging.StreamHandler",
+                "formatter": "simple",
+            },
+        },
+        "root": {"handlers": ["console"], "level": "WARNING"},
+        "loggers": {
+            "django": {
+                "handlers": ["console"],
+                "level": "WARNING",
+                "propagate": False,
+            },
+            "django.request": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+            "django.channels.server": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+            "aap_eda": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+            "ansible_base": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+        },
+    }
+
+
+def _get_default_pg_notify_dsn_server(settings: Dynaconf) -> str:
+    db_options = settings.DATABASES["default"].get("OPTIONS", {})
+    return (
+        f"host={settings.DATABASES['default']['HOST']} "
+        f"port={settings.DATABASES['default']['PORT']} "
+        f"dbname={settings.DATABASES['default']['NAME']} "
+        f"user={settings.DATABASES['default']['USER']} "
+        f"password={settings.DATABASES['default']['PASSWORD']} "
+        f"sslmode={db_options.get('sslmode','allow')} "
+        f"sslcert={db_options.get('sslcert','')} "
+        f"sslkey={db_options.get('sslkey','')} "
+        f"sslrootcert={db_options.get('sslrootcert','')} "
+    )
+
+
+def _set_resource_server(settings: Dynaconf) -> None:
+    if (
+        settings.RESOURCE_SERVER["URL"]
+        and settings.RESOURCE_SERVER["SECRET_KEY"]
+    ):
+        jobs = settings.RQ_PERIODIC_JOBS
+        jobs.append(
+            {
+                "func": "aap_eda.tasks.shared_resources.resync_shared_resources",  # noqa: E501
+                "interval": 900,
+                "id": "resync_shared_resources",
+            }
+        )
+        settings.RQ_PERIODIC_JOBS = jobs
+
+
+def post_loading(settings: Dynaconf):
+    settings.SECRET_KEY = _get_secret_key(settings)
+    settings.DEBUG = get_boolean(settings, "DEBUG")
+    settings.ALLOWED_HOSTS = _get_list_from_str(settings, "ALLOWED_HOSTS")
```

Comment:
It is ok to mutate the `settings` object directly here like `settings.FOO = "BAR"`

However, those changes will not be tracked when running `dynaconf inspect` commands.

I would suggest using dictionaries to aggregate settings to be mutated.

example:

```py

def post_loading(settings: Dynaconf):
    data = {}
    data["FOO"] = "BAR"
    data["THING"] = value

    # then finally
    settings.update(data, loader_identifier="post_loading")

```

The `loader_identifier` is just a unique string to be shown when running `dynaconf inspect -k FOO` so when troubleshooting we know exactly where those values came from.